### ISSUE TITLE
Search Function Not Implemented

### ISSUE THREAD

**Issue by FlamesRunner** [FlamesRunner/natCP#5]
The search function within the administrator panel has not yet been implemented.
If you'd like to have a try at it, fork the project and make the appropriate changes, then submit a pull request.

It should be simple; all you need to do is add a section to display search results (e.g foreach rows), and implement the database query.

You can easily access the database object with $dbh->(pdo functions).

**Comment by hydeenoble** [comment]
I would like to work on this. please, assign the issue to me

**Comment by FlamesRunner** [comment]
It's all yours, but I cannot assign the issue to you without giving full access. 
Sorry about that!

**Comment by mikebywater** [comment]
Is this a search just for the virtual servers? I have implemented a js search on the users in the refactor branch  (its more of a filter) which will scale to a certain degree, I can test how far, if it seems tidy enough I will implement something similar when I refactor the server section

**Comment by FlamesRunner** [comment]
Yes. The initial query that was used was 'SELECT * FROM virtualservers where v4addresses LIKE (search query);'.

However, it was not reimplemented from the first version.

**Comment by FlamesRunner** [comment]
The issue is that the panel uses SSH connections -- and it's rather inefficient. In the next week or so, I will be reimplementing the slave node system to host a web API so that a simple cURL request will do it.

**Comment by FlamesRunner** [comment]
I am mentioning this as it will bottleneck the searching process.
